frame/support/src/traits/try_runtime.rs

```diff
@@ -115,21 +117,20 @@ impl sp_std::str::FromStr for Select {
 	}
 }
 
-/// Execute some checks a pallet to ensure the state is consistent.
+/// Execute some checks to ensure the internal state of a pallet is consistent.
+///
+/// Similar
+///
+/// Usually, these checks should check all of the invariants that are expected to be help on all of
+/// the storage items of your pallet.
```

Comment:
I don't understand what the part about "to be help" means. Rephrase?

Reply:
I meant "held" 🤦 sorry